.gitmodules

```diff
@@ -4,7 +4,7 @@
 [submodule "NuttX"]
 	path = NuttX
 	url = git://github.com/PX4/NuttX.git
-[submodule "libuavcan"]
+[submodule "src/lib/uavcan"]
```

Comment:
OK, this is not related to the fix. I has to change this in order to successfully update the submodule. Without the changes Tortoisegit was not able to update this submodule on my windows machine.